crypto/src/signer.rs

```diff
@@ -2,7 +2,8 @@
 
 use crate::{AlgLen, CryptoError};
 
-/// An ECDSA signature
+/// A cryptographic signature representing an ECDSA signature or an HMAC.
```

Comment:
nit: We should probably leave out the HMAC comment here. It doesn't really apply to the `crypto` library, just the way we use it in `dpe`.